### DISCUSSION TITLE
I can't acces to the webpart as guest user

### DISCUSSION THREAD

**Discussion by Souhailbrs** [microsoft-search/pnp-modern-search#2640] [Q&A]
Hello ,
i had an error that actually saying that something wrong and i tried all solutions thet you provided but still not working!
it work once if i open the site from invitaion link please help asap

![Capture](https://user-images.githubusercontent.com/59336158/206225371-dcb8549d-75bf-459f-88dc-51436ce8c99c.PNG)

**Comment by wobba** [comment]
See this post by @LauraKokkarinen which explains SPFx and guest user setup. https://laurakokkarinen.com/sharepoint-online-guest-user-troubles-and-how-to-get-past-them/
